### PR TITLE
Bug - cpp11 source file names

### PR DESCRIPTION
**Pull request by sbearrows**
This should fix the R CMD check errors and still prevent source file name changes.

### CODE REVIEW

File: tests/testthat/test-source.R [hunk 1]
```diff
@@ -101,3 +101,7 @@ test_that("generate_include_paths handles paths with spaces", {
     expect_equal(generate_include_paths("cpp11"), "-I'/a path with spaces/cpp11'")
   }
 })
+
+test_that("", {
+
```
Comment (jimhester) [inline tests/testthat/test-source.R:106]: Are we missing the tests for this?

Comment (jimhester) [reply tests/testthat/test-source.R:106]: Also 🙏 in advance for writing a test for it!

Comment (sbearrows) [reply tests/testthat/test-source.R:106]: Wow I'm sooo good at writing tests :satisfied:

**Comment by jimhester** [comment]
Thanks!

**Pull request merged**